sdk/keyvault/ci.yml

```diff
@@ -1,6 +1,14 @@
 # DO NOT EDIT THIS FILE
 # This file is generated automatically and any changes will be lost.
 
+schedules:
+- cron: "0 0 * * *"
```

Comment:
What does this mean?

Reply:
It is supposed to be cron format. So this means run once per day, at midnight. Translated from left to right:

At the zeroth minute
Of the zeroth hour
Of every day
Of every month
Including MTWTFSS

This might help:

https://crontab.guru/